### PR TITLE
26 cli

### PR DESCRIPTION
**Pull request by artog**

### CODE REVIEW

File: src/main/java/ui/Main.java [hunk 1]
```diff
@@ -30,6 +34,12 @@
  */
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello World");
+        CLI cli = new CLI(new File(""));
+        cli.start();
+    }
+    
+    
+    public static IAutomaton buildAutomata() {
```
Comment (artog) [inline src/main/java/ui/Main.java]: Ops, legacy

**Comment by artog** [comment]
Currenty supporting commands (some not implemented):
- quit / q
- print / p, print table(+ more?) of currently loaded automata
- load / l <file>, loads automata from file.
- reload / r, reload the currently loaded file (e.g. if something was changed)

All commands starts with a colon, a la ghci.

**Comment by artog** [comment]
Is now a working program. Can:
- run program: `java -jar target/automatabuilder-1.0.0-SNAPSHOT.jar`
- load file from argument `java -jar target/automatabuilder-1.0.0-SNAPSHOT.jar file.xml`
- print table using :p 
- test input directly by typing in command line
- load and reload new/same file

**Comment by Oliv95** [comment]
Looks good to me :+1:

**Pull request merged**